docs/programming-guide.md

```diff
@@ -24,7 +24,8 @@ along with if you launch Spark's interactive shell -- either `bin/spark-shell` f
 
 <div data-lang="scala"  markdown="1">
 
-Spark {{site.SPARK_VERSION}} uses Scala {{site.SCALA_BINARY_VERSION}}. To write
+Spark {{site.SPARK_VERSION}} is built and distributed to work with Scala {{site.SCALA_BINARY_VERSION}} 
+by default. (Spark can be built to work with other versions of Scala, too.) To write
```

Comment:
This makes it sound like Spark is only built for 2.10 and you have to build it yourself to use 2.11. But that's not right is it? I think Spark is built for both 2.10 and 2.11